lib/app/views/dashboard.erb

```diff
@@ -15,13 +17,28 @@
           <% if current_user.ongoing.any? %>
             <p>Keep working on these problems, or mark them as done.</p>
             <ul class="nav nav-stacked" id="menu-current" role="menu">
-              <% current_user.ongoing.each do |submission| %>
+              <% current_user.ongoing[0..4].each do |submission| %>
```

Comment:
`take` is an array method. It's the same as `first`:

```
(1..15).to_a.first(3)
(1..15).to_a.take(3)
```

Try opening bin/console and using `to_sql` for those queries:

```
> u = User.first
> u.ongoing.to_sql
> u.ongoing.limit(5).to_sql
```

Now imagine that the user has a million current exercises.